tests/tests/test_IO/test_cell_io.py

```diff
@@ -55,91 +48,210 @@ def z_vals(data_path):
 
 
 @pytest.fixture
-def cubes_cells():
+def cells_with_artifacts():
     return [
-        Cell([340, 1004, 15], 1),
-        Cell([340, 1004, 15], 1),
-        Cell([392, 522, 10], 1),
+        Cell([340, 1004, 15], -1),
+        Cell([340, 1004, 15], -1),
```

Comment:
These -1's have gone to +1s just because of the new tests that we're adding? Or has there been an actual functionality change?

Reply:
Yes! I moved the original `cubes_cells` and `roi_sorter_cells` fixtures into the `parametrize` call of `test_get_cells_dir`, as they were only used in that one test function. I also thought it made it a bit clearer that these are the expected contents of the test data under `/cube_extract/cubes` and `/IO/roi_sorter_output`.

The `cells_with_artifacts` fixture was added for the new tests that handle cells with type -1 (`test_cells_to_xml_artifacts_keep` and `test_get_cells_xml_cells_only`).